### PR TITLE
G17: Add product definition

### PR DESCRIPTION
**Pull request by kodawah**

### CODE REVIEW

File: data/contents/G17.yaml [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
-  2017 Gift Pack: []
+  2017 Gift Pack:
+    card_count: 7
+    - card:
+      foil: true
+      name: Plains
+      number: 1
+      set: g17
+    - card:
+      foil: true
```
Comment (axxroytovu) [inline data/contents/G17.yaml:11]: Card is a top level key. Should be:
```
card:
- foil: true
  name: Plains
  ...
- foil: true
  name: Mountain
  ...
```

**Comment by axxroytovu** [comment]
The GHA check for a pull request shows if you formatted your yaml correctly. If the check fails you should automatically look for where your yaml is wrong.

**Review by axxroytovu — CHANGES_REQUESTED** [review]

File: data/contents/G17.yaml (hunk 1 above)
Comment (kodawah) [reply data/contents/G17.yaml]: mb, fixed

**Comment by kodawah** [comment]
I'm very glad for the GHA automation, but man i find yaml syntax SO odd 😅

**Review by axxroytovu — APPROVED** [review]

**Pull request merged**